ui/tests/acceptance/actions-test.js

```diff
@@ -165,36 +167,74 @@ module('Acceptance | actions', function (hooks) {
     // run on a random alloc
     await Actions.titleActions.multiAllocActions[0].subActions[0].click();
 
+    assert.ok(Actions.flyout.isPresent);
     assert.equal(
-      Actions.toast.length,
+      Actions.flyout.instances.length,
       1,
-      'A toast notification pops up upon running an action'
+      'A sidebar instance pops up upon running an action'
     );
 
     assert.ok(
-      Actions.toast[0].code.includes('Message Received'),
-      'The notification contains the message from the action'
+      Actions.flyout.instances[0].code.includes('Message Received'),
+      'The instance contains the message from the action'
     );
     assert.ok(
-      Actions.toast[0].titleBar.includes('Finished'),
-      'The notification contains the message from the action'
+      Actions.flyout.instances[0].statusBadge.includes('Complete'),
+      'The instance contains the status of the action'
     );
 
-    // run on all allocs
+    await Actions.flyout.close();
+    // Type the escape key: the Helios dropdown doesn't automatically close on click-away events
+    // as defined by clickable in the page object here, so we should explicitly make sure it's closed.
+    await triggerEvent('.job-page-header .actions-dropdown', 'keyup', {
+      key: 'Escape',
+    });
+
+    assert.notOk(Actions.flyout.isPresent);
+    assert.equal(Actions.titleActions.expandedValue, 'false');
+
+    await Actions.titleActions.click();
+    await Actions.titleActions.multiAllocActions[0].button[0].click();
     await Actions.titleActions.multiAllocActions[0].subActions[1].click();
 
+    assert.ok(Actions.flyout.isPresent);
+
+    // assert.equal(
```

Comment:
could we perhaps remove this? I live by the rule that commented out code doesn't belong on `main` unless there's a good reason for it.

Reply:
good catch!